source/includes/_tipser-elements.md

```diff
@@ -169,14 +169,15 @@ All configuration supported by Tipser Elements is listed below.
 
 Parameter | Default | Description | Example
 --------- | ------- | ----------- | -------
-lang | `'en'` | a locale to be used by the Tipser content. Possible values: `'en'`, `'de'`, `'fr'` and `'sv'`. More info at [Language and locale](#language-and-locale)[Environment](#environment)| `'de'` 
+lang | `'en-US'` | a locale to be used by the Tipser content. Possible values: `'en-US'`, `'de-DE'`, `'fr-FR'` and `'sv-SE'`. More info at [Language and locale](#language-and-locale)[Environment](#environment)| `'de-DE'` 
```

Comment:
Currently there is no difference between locale after your changes and before, in terms of how our tipser-elements works. Lang for three things:
- set lang to webtranslate helper 
- send locale to contentful
- send defaultLang to iframe content

All this works okay with "before" your changes. Moreover -> default value is actually `"en"` not `"en-US"`. https://github.com/Tipser/tipser-elements/blob/0fc20acf5d326831e1ae117fc10b43819963a9a4/library/tipser-context.tsx#L76

We have a bit of mess there I think. What is the motivation of your changes ?

Reply:
Piotr asked me to change this because this is the direction we are heading. It's good you mentioned TE defaults because we need to change them in order to make the projects in TE & CF & FE consistent.

Reply:
@chmurson this documentation change it's a consequence of a longer discussion including Jonas and @mateuszsikora . You are right that the short versions of locales work mostly okay as for now. However there have been inconsistencies between FE and Elements how we name locales that forced @mateuszsikora to introduce some special mapping logic.

So the question raises: can we just stick to one standard of naming locales and which one would it be? We think that simply using full locale names everywhere will be the most future proof, and least error-prone, especially that we've had in the past the case of Swiss (swiss french vs swiss german) and still consider returning to that market.

Reply:
@piotr-sobczyk makes sense to me. We should schedule changes in tipser-elements accordingly as well.

Reply:
@chmurson you're right, I added a technical task for it: https://tipser.atlassian.net/browse/T3-3950 .